src/Giraffe/ResponseCaching.fs

```diff
@@ -55,14 +55,17 @@ module ResponseCaching =
             | Public duration -> tHeaders.CacheControl <- cacheHeader true duration
             | Private duration -> tHeaders.CacheControl <- cacheHeader false duration
 
-            if vary.IsSome then
-                headers.[HeaderNames.Vary] <- StringValues [| vary.Value |]
+            match vary with
```

Comment:
Option.iter maybe?

Reply:
Indeed, using `Option.iter` improves readability (my POV), thanks for the suggestion!